scripts/talos/bumpSimorghPackages/getUpdates/index.js

```diff
@@ -0,0 +1,20 @@
+const semver = require('semver');
+
+const getUpdates = (dependencies, publishedPackages) => {
+  return Object.keys(publishedPackages)
+    .filter(packageName => {
+      const oldVersion = dependencies[packageName];
+      const newVersion = publishedPackages[packageName];
+      return (
+        semver.validRange(oldVersion) &&
+        semver.validRange(newVersion) &&
+        semver.intersects(oldVersion, newVersion)
+      );
+    })
+    .map(
+      packageName =>
+        `${packageName}  ${dependencies[packageName]}  →  ${publishedPackages[packageName]}`,
```

Comment:
Is this being done to mimic the output of `npm-check-updates`? why?

Reply:
Yes it is. Because there is a function written to take advantage of that structure. If we effectively take `npm-check-updates out`, we can do without the mimicking and have a unified structure across.

Reply:
Talos was written to update Psammead. With no intention of updating another repository. The current features of Talos like running commit messages and pushes from the command line will mean that we will have to clone the entire Simorgh repository and isolate it from the current git repository. Which seems like a lot of work with a lot of instability.

Reply:
A lot of the new functions written can replace the current way Talos runs by using Github API to make commits and pushes. That will unify both methods.